packages/plugin-rest-client-shared-cache/gulpfile.js

```diff
@@ -0,0 +1,52 @@
+require('babel-core/register.js')({
```

Comment:
change to https://github.com/seznam/IMA.js-plugins/blob/master/packages/plugin-useragent/gulpfile.js